ansible/files/prometheus/prometheus.yml

```diff
@@ -21,6 +21,58 @@ scrape_configs:
         - 'tradedefecit.conference.fosdem.net:9100'
         - 'prometheus.conference.fosdem.net:9100'
 
+  - job_name: 'video_node'
```

Comment:
I wanted to group them separately, but I guess it's fine to just put them in the node group.